kartothek/io_components/docs.py

```diff
@@ -65,8 +65,9 @@
     "partition_on": """
     partition_on: list
         Column names by which the dataset should be partitioned by.
```

Comment:
```suggestion
         Column names by which the dataset should be partitioned by physically.
```

This makes it clearer that this is not just an index structure.